mlflow/tracing/__init__.py

```diff
@@ -0,0 +1,172 @@
+from collections import defaultdict
+from typing import TYPE_CHECKING, Any, Dict, List, Literal, NamedTuple, Optional, Union
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.tracing.utils import SPANS_COLUMN_NAME, traces_to_df
+
+if TYPE_CHECKING:
+    import pandas as pd
+
+    import mlflow.entities
+
+
+def extract(
```

Comment:
nit: This only seems to be used by `search_traces()` method, can we move it under `fluent.py` and private?